save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/VulnerabilityFilters.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.filters
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property prefixName
+ */
+@Serializable
+data class VulnerabilityFilters(
```

Comment:
can we rename it to `VulnerabilityFilters`? It's a single filter

Reply:
Yes, sure

Reply:
Suggestion to leave as is (VulnerabilityFilters) in order to keep consistence

Reply:
we have a task to rename all `Filters` to `Filter` -- no need to produce more

Reply:
Renamed all Filters to Filter in project